### ISSUE TITLE
Missing indirect crossrefs cause 'Can't call method "get_field" on an undefined value'

### ISSUE THREAD

**Issue by logological** [plk/biber#115]
When someone cites a BibTeX entry containing a crossref field, and the ID of that field does not match the key of any other BibTeX entry, then Biber correctly outputs the following helpful warning message:

    WARN - I didn't find a database entry for crossref 'foo' in entry 'bar' - ignoring (section 0)

However, this warning message does not appear when the missing crossref is at the end of a longer chain of crossrefs.  Instead, Biber quits with a cryptic and unhelpful error message.  For example, if the entry `gnu` has a crossref to the existing entry `gnuconf2016`, but `gnuconf2016` itself has a crossref to a non-existing entry `gnuproceedingsx`, then Biber instead fails with the following error:

    Can't call method "get_field" on an undefined value at /tmp/par-6d696c6c6572/cache-ab2aa2ece61cc988b50d107b05c5c3fad9d00256/inc/lib/Biber.pm line 3377.

Below is a minimal example reproducing the problem.  Here the entry `gnuconf2016` has a mistyped crossref to `gnuproceedingsx` instead of `gnuproceedings`.  I think the use of indirect references is not particularly uncommon; a user may have several `@inproceedings` entries that are crossreferenced to the same `@proceedings` entry, and then multiple `@proceedings` entries that get their publisher information from the same entry via a crossref.

```
\begin{filecontents}{\jobname.bib}
@proceedings{gnuproceedings,
  publisher = {Free Software Society},
  address = {Cambridge, MA},
  dataonly = {true},
}

@proceedings{gnuconf2016,
  title = {30th Annual Meeting of the FSF},
  year = 2016,
  crossref = {gnuproceedingsx},
}

@inproceedings{gnu,
  title = {Gnus I Have Known and Loved},
  author = {Gary Gnu},
  crossref = {gnuconf2016},
}
\end{filecontents}
\documentclass{article}
\usepackage[backend=biber]{biblatex}
\bibliography{\jobname}
\begin{document}
\textcite{gnu}
\printbibliography
\end{document}
```

Possibly this is related to Issue #93.

I am using Biber 2.4 with biblatex 3.3 (2016/03/03).

**Comment by plk** [comment]
Coincidentally, I think this was just fixed in the 2.5 DEV version which you can find on Sourceforge (see biblatex ticket (https://github.com/plk/biblatex/issues/395). Can you try this? You will need to be using biblatex 3.4 dev version from SourceForge too.

**Comment by logological** [comment]
Yes, when using the development versions of biblatex 3.4 and biber 2.5, I get the "I didn't find a database entry for crossref" warning as expected instead of the cryptic "Can't call method 'get_field'" error.  Thanks for the information.
